solr/core/src/java/org/apache/solr/handler/admin/ConfigSetsHandler.java

```diff
@@ -166,31 +171,96 @@ private void handleConfigUploadRequest(SolrQueryRequest req, SolrQueryResponse r
     InputStream inputStream = contentStreamsIterator.next().getStream();
 
     // Create a node for the configuration in zookeeper
-    boolean trusted = getTrusted(req);
-    zkClient.makePath(configPathInZk, ("{\"trusted\": " + Boolean.toString(trusted) + "}").
-        getBytes(StandardCharsets.UTF_8), true);
+    Set<String> filesToDelete = Collections.emptySet();
```

Comment:
this entire "filesToDelete" is likely an error made when creating a backport patch for it, this set is never populated, so it has no chance to do anything. I can remove lines related to it I guess